RELEASE.md

```diff
@@ -4,6 +4,7 @@
 ## Bug fixes and other changes
 * Fixed bug where project creation workflow would use the `main` branch version of `kedro-starters` instead of the respective release version.
 * Fixed namespacing for `confirms` during pipeline creation to support `IncrementalDataset`.
+* Add deprecation warning for `namespace`.
```

Comment:
```suggestion
* Add deprecation warning for `--namespace` option for `kedro run`. It will be replaced with `--namespaces` option which will allow for running multiple namespaces together.
```

Reply:
Just a suggestion for a more descriptive note.

Reply:
I've actually added a new header in [my PR](https://github.com/kedro-org/kedro/pull/4653/files) for this type of change: ## Upcoming deprecations for Kedro 1.0.0

In case your PR gets merged first, just add the header here.